maxim/logger/together/client.py

```
@@ -0,0 +1,224 @@
+                )
+
+            return response
+
+        return wrapper
+
+    def wrap_async_create(create_func):
+        '''Wrapping the create function of chat completion for the asynchronous client'''
+
+        @functools.wraps(create_func)
+        async def wrapper(self: AsyncChatCompletions, *args: Any, **kwargs: Any):
+            extra_headers = kwargs.get("extra_headers", None)
```

Ruff (0.11.9)

Use `kwargs.get("extra_headers")` instead of `kwargs.get("extra_headers", None)`

Replace `kwargs.get("extra_headers", None)` with `kwargs.get("extra_headers")`

(SIM910)